src/processors/processor.ts

```diff
@@ -59,6 +61,8 @@ export default class Processor {
    * @returns The generated module classname
    */
   public createModuleClassname = (name: string): string => {
+    if (!this.ast.css) throw new Error('ast.css is not defined.');
```

Comment:
Hack: Needed to do this to pass linting since the type of `Ast.css` changed to be nullable.